SPS_BACK-UP_RCS_CONTROL.s

```diff
@@ -114,7 +114,7 @@ NEGCHECK	INDEX	AXISCTR		# JETS FIRING NEGATIVELY
 		TCF	+2
 		TCF	+1		# JETS COMMANDED OFF.  SET CTR AND RETURN
 SETCTR		INDEX	AXISCTR		# JET FIRING REVERSAL COMMANDED.  SET CTR,
-		CA	UTIME		# SET JET TIME TO ZER, AND RETURN
+		CA	UTIME		# SET JET TIME TO ZERO, AND RETURN
```

Comment:
Correct, but needs space removing after `ZERO,` - i.e `ZERO,AND RETURN`. [Source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1508.jpg)